internal/acceptance/share_test.go

```diff
@@ -93,3 +105,54 @@ func TestUcAccCreateShare(t *testing.T) {
 		`,
 	})
 }
+
+func TestUcAccCreateUpdate(t *testing.T) {
```

Comment:
The test name must say something about which component is being tested.

Reply:
Done